Dockerfile

```diff
@@ -1,25 +1,41 @@
-FROM centos:7
+FROM registry.access.redhat.com/ubi8/ubi-minimal
 MAINTAINER Pavlina Bortlova <pbortlov@redhat.com>
 
-LABEL description="Review-rot - gather information about opened merge or pull requests"
-LABEL summary="review-rot git gitlab github pagure gerrit"
-LABEL vendor="PnT DevOps Automation - Red Hat, Inc."
+LABEL \
+    description="Review-rot - gather information about opened merge or pull requests" \
+    summary="review-rot git gitlab github pagure gerrit" \
+    vendor="PnT DevOps Automation - Red Hat, Inc."
```

Comment:
sorry for bothering: could you plase change the vendor? to Process-tools guild? https://docs.engineering.redhat.com/display/EXD/reviewrot

Reply:
Changed, but not sure which format would you prefer.